turbopack/crates/turbopack-tests/tests/execution.rs

```diff
@@ -462,13 +463,9 @@ async fn run_test_operation(prepared_test: ResolvedVc<PreparedTest>) -> Result<V
         )
         .module();
 
-    // Keep this in sync with what `evaluate` does internally
-    let module_graph = ModuleGraph::from_modules(
-        Vc::cell(vec![ChunkGroupEntry::Entry(vec![
-            jest_entry_asset.to_resolved().await?,
-        ])]),
-        false,
-    );
```

Comment:
This is the hack that was used previously (hoping that this module graph, and the one created in `evaluate` below would be identical or equivalent).